corehq/apps/api/resources/v0_4.py

```diff
@@ -120,6 +120,7 @@ def obj_get_list(self, bundle, domain, **kwargs):
 
         def wrapper(doc):
             if doc['doc_type'] in xform_doc_types:
+                doc.pop('user_id', None)  # needed when xform is stored in sql
                 return xform_doc_types[doc['doc_type']].wrap(doc)
```

Comment:
i think this is more thoroughly broken then we think. right now what's happening is when the sql form gets moved to json, it adds a doc_type property that tricks this "wrapper" into thinking it's couch and then we try and wrap the sql doc in a couch class. XFormInstanceSQL doesn't have a wrap class